.github/workflows/golangci-lint.yaml

```diff
@@ -51,6 +51,6 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-         go-version: "1.21.5"
+          go-version: "1.21.9"
```

Comment:
It's nice that we can update the z-version from .5 to .9 and don't have to change the .21 y-version.

We could potentially fix future changes also by using `go-version: "1.21"` here without the z-version . We did a similar thing recently in Rook upstream here: https://github.com/rook/rook/pull/14175/files